### DISCUSSION TITLE
Please enable Trojan-Go!

### DISCUSSION THREAD

**Discussion by essoojay** [hiddify/Hiddify-Manager#511] [General سایر]
Hi, please consider enabling Trojan-Go, its disable by default and its specified to be disabled on github.

Thanks

**Comment by zZedix** [comment]
unfortunately trojan-go doesn't have so many client software across multi platforms !

**Reply by essoojay** [reply]
It does, Sagernet, Matsuri, Nekobox for Android, Clash for Android all for Android.  IOS has shadowrocket, Stash etc, and Windows Desktop has Nekoray, V2rayNG, Netch, Clash for Windows and many more. so whats your point?
